services/deposit/deposit.go

```diff
@@ -189,7 +190,10 @@ func Run(config *Config) {
 			Info("Failed to load state, creating empty state")
 		state = &runState{}
 		blockNumber := eth.GetHeight(config.LoadBalancer)
-		state.LastEthBlock = blockNumber - config.BlockDelay
+		state.LastEthBlock = config.StartFromBlock + config.BlockDelay
```

Comment:
Should mark `LastConfirmedBlock` so changing `config.BlockDelay` will not miss blocks